extensions/wikia/Recirculation/services/DiscussionsDataService.class.php

```diff
@@ -46,10 +46,17 @@ private function apiRequest() {
 		$endpoint = $wgCityId . '/forums/' . $wgCityId;
 
 		$url = $this->buildUrl( $endpoint, $options );
-		$data = Http::get( $url );
 
-		$obj = json_decode( $data, true );
-		return $obj['_embedded']['doc:threads'];
+		try {
+			$data = Http::get( $url );
```

Comment:
Generally yes, however it seems to throw exceptions when the route times out as per: https://wikia-inc.atlassian.net/browse/RECIRC-106. I could most like just wrap the `Http::get` in a try catch block, but seemed to make sense just to put it all in here even if the rest of the methods don't throw exceptions